### PR TITLE
Rambo Survivors

### PR DESCRIPTION
**Pull request by Isy232**
## About The Pull Request

This is in general a large buff to survivors, they start with decent armor and master in all skills except combat ones.

If you want an IC explanation, consider why you, out of everyone in the colony, is still alive while the others are dead. You must be really lucky or really good.

## Why It's Good For The Game

Survivor is masochistic already, why not give them a little something to slightly increase their chance of making it out alive as well as giving the marines a reason to rescue survivors.

Also, lots of PR's, some I've made, have side effects of making survivors harder, this should help to counter that.

## Changelog
:cl:
balance: Survivor gets master in all non-combat skills.
balance: Survivor gets decent armor.
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -3. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by spookydonut** [comment]
> 
> 
> You currently have a negative Fix/Feature pull request delta of -3. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

File: code/datums/jobs/job/survivor.dm [hunk 1]
```diff
@@ -24,7 +24,6 @@
 //Scientist
 /datum/job/survivor/scientist
 	title = "Scientist"
-	skills_type = /datum/skills/civilian/survivor/scientist
```
Comment (Rohesie) [inline code/datums/jobs/job/survivor.dm]: Keep this.

File: code/datums/jobs/job/survivor.dm [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /datum/job/survivor
 	access = list(ACCESS_CIVILIAN_PUBLIC, ACCESS_CIVILIAN_RESEARCH, ACCESS_CIVILIAN_ENGINEERING, ACCESS_CIVILIAN_LOGISTICS)
 	minimal_access = list(ACCESS_CIVILIAN_PUBLIC, ACCESS_CIVILIAN_RESEARCH, ACCESS_CIVILIAN_ENGINEERING, ACCESS_CIVILIAN_LOGISTICS)
-	skills_type = /datum/skills/civilian/survivor
```
Comment (Rohesie) [inline code/datums/jobs/job/survivor.dm]: Keep his, just make another variant.

File: code/datums/jobs/job/survivor.dm [hunk 3]
```diff
@@ -218,3 +218,23 @@
 	wear_suit = /obj/item/clothing/suit/storage/CMB
 	shoes = /obj/item/clothing/shoes/jackboots
 	back = /obj/item/storage/backpack/satchel/sec
+
+
+
+// Rambo Survivors
+
+//Generic
+/datum/job/rambosurvivor/generic
```
Comment (Rohesie) [inline code/datums/jobs/job/survivor.dm]: Is there a non-genric rambosurvivor?

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-4). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Comment by jroinc** [comment]
OOF MERGED A SPELLING ERROR.

RuggeT boots?

**Pull request merged**